CustomScript/customscript.py

```diff
@@ -594,5 +592,20 @@ def get_formatted_log(summary, stdout, stderr):
     return msg_format.format(summary, stdout, stderr)
 
 
+def get_command_to_execute(hutil):
+    public_settings = hutil.get_public_settings()
+    protected_settings = hutil.get_protected_settings()
+    cmd_public = public_settings.get('commandToExecute')
```

Comment:
can we make 'commandToExecute' a constant string instead of hard code several times.